app/src/main/java/com/example/bookwormadventuresdeluxe2/ViewLocationActivity.java

```diff
@@ -6,19 +6,36 @@
 
 import androidx.appcompat.app.AppCompatActivity;
 
-public class ViewLocationActivity extends AppCompatActivity
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.SupportMapFragment;
+
+public class ViewLocationActivity extends AppCompatActivity implements OnMapReadyCallback
 {
     TextView textViewAddress;
     Button scanISBNButton;
 
+    GoogleMap map;
+    SupportMapFragment mapFragment;
+
     @Override
     protected void onCreate(Bundle savedInstanceState)
     {
         setTitle("View Location");
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_view_location);
 
-        textViewAddress = (TextView) findViewById(R.id.view_address_text);
-        scanISBNButton = (Button) findViewById(R.id.scan_isbn_button);
+        textViewAddress = findViewById(R.id.view_address_text);
+        scanISBNButton = findViewById(R.id.scan_isbn_button);
+
+        mapFragment = (SupportMapFragment) getSupportFragmentManager().findFragmentById(R.id.google_map);
+        mapFragment.getMapAsync(ViewLocationActivity.this);
     }
+
+    @Override
+    public void onMapReady(GoogleMap googleMap) {
```

Comment:
config

Reply:
yea i have the macros. dunno how i missed that though

Reply:
fixed